src/components/PopupConfirm.js

```diff
@@ -6,16 +6,23 @@ export default class PopupConfirm extends Popup {
     this._confirmDeleteButton = this._modal.querySelector(
       ".modal-confirm__submit-button"
     );
+    this._submitButton = this._modal.querySelector(".modal__submit-button");
   }
 
   handleConfirm(action) {
     this._handleConfirmDelete = action;
   }
 
+  handleButtonLoading() {
+    this._submitButton.textContent = "Saving...";
+    setTimeout(() => {
+      this._submitButton.textContent = "Save";
```

Comment:
* When I delele a card the button text becomes `Save` instead of `Yes` https://disk.yandex.ru/i/iP8IMmR0QRctOA